src/gluonts/dataset/repository/_lstnet.py

```diff
@@ -174,16 +174,16 @@ def generate_lstnet_dataset(dataset_path: Path, dataset_name: str):
 
     # time of the first prediction
     prediction_dates = [
-        training_end + i * ds_info.prediction_length
+        training_end + (i * ds_info.prediction_length) * training_end.freq
```

Comment:
There is [`frequency_add`](https://github.com/awslabs/gluon-ts/blob/95201b0bbbafd7d8a618217e64a0cae6b6eaf025/src/gluonts/support/pandas.py#L18)

Maybe we can use it, or something similar instead?

Reply:
true! forgot about that, fixing